### ISSUE TITLE
Tab color disappeared after updates

### ISSUE THREAD

**Issue by imaginois** [gpakosz/.tmux#282]
The background color of tabs disappeared after I updated, tmux and item2. I really don't know what the main reason might be because I updated multiple versions at the same time.

Here is a screenshot

<img width="848" alt="Screen Shot 2019-09-18 at 15 48 30" src="https://user-images.githubusercontent.com/685519/65155131-faa45280-da2c-11e9-8171-335d045a739e.png">

I've updated to the latest __.tmux.conf__ and __.tmux.conf.local__

**Comment by gpakosz** [comment]
Are you sure you upgraded to the latest version of this conf?

Looks like a duplicate of #237 to me which has been fixed by
```
commit e1452f124247120f3c8f27411d062b81d9919fe2
Author: Gregory Pakosz <gregory.pakosz@gmail.com>
Date:   Thu Apr 25 09:29:48 2019 +0200

    worked around tmux 2.9 "style inheritance" behavior change, fixes #237
```

**Comment by imaginois** [comment]
Yep, my bad. The files were identical, but after looking at #237 I figured out I have to reload the configuration. I though killing tmux-server did that, but I guess not

Make sure new conf is applied, guys :)

Closing
